install/dc-detect-version.sh

```diff
@@ -1,3 +1,11 @@
+if [ "${GITHUB_ACTIONS:-}" = "true" ]; then
```

Comment:
> Yeah, we need to rebase/update.

@chadwhitacre I mean is there some environment variable or something like that in GCB which should be checked like here?

Reply:
This can stay as-is, only GHA parses the `::` in a special way (right?), other envs can all use the arrow output.